langchain/src/util/websocket_stream.ts

```diff
@@ -0,0 +1,101 @@
+import WsWebSocket from "ws";
```

Comment:
I like this but can we leave it inside the `iflytek` class for now?

Reply:
yes, I will change this name. WebsocketStream is a explainer to chrome, but is not launch. I think WebSocketStream is a good design, I do it base on these:
[websocketstream-explainer](https://github.com/ricea/websocketstream-explainer/blob/master/README.md)
[chrome developer current status](https://developer.chrome.com/articles/websocketstream/)